tests/ConnectionTest.php

```diff
@@ -482,4 +483,23 @@ protected function performRollBack($toLevel): void
         self::assertNull($conn->getCurrentTransaction());
         self::assertSame(0, $conn->transactionLevel());
     }
+
+    public function test_escape(): void
+    {
+        $conn = $this->getDefaultConnection();
+
+        self::assertSame('true', $conn->escape(true));
+        self::assertSame('false', $conn->escape(false));
+        self::assertSame('1', $conn->escape(1));
+        self::assertSame('0', $conn->escape(0));
+        self::assertSame('-1', $conn->escape(-1));
+        self::assertSame('1.1', $conn->escape(1.1));
+        self::assertSame('"a"', $conn->escape('a'));
+        self::assertSame('r"""' . "\n" . '"""', $conn->escape("\n"));
+        self::assertSame('[]', $conn->escape([]));
+        self::assertSame('["a"]', $conn->escape(['a']));
+        self::assertSame('[false]', $conn->escape([false]));
+        self::assertSame('[1]', $conn->escape([1]));
+        self::assertSame('[1.1]', $conn->escape([1.1]));
```

Comment:
```suggestion
        self::assertSame('[1.1]', $conn->escape([1.1]));
        self::assertSame('[1, [2, [3]]]', $conn->escape([1, [2, [3]]]));
```

Shouldn't multidimensional arrays also be included in the assertion to test the circular process?

Reply:
multi dimensional arrays are not supported.

https://cloud.google.com/spanner/docs/reference/standard-sql/data-types#declaring_an_array_type

Reply:
Oops... It seems to pass the test case for multidimensional arrays, so I guess we need to fix it so it doesn't pass?

Reply:
That should be the expected behavior. This has been fixed.